src/Cli/dotnet/CommandFactory/CommandResolution/DepsJsonCommandResolver.cs

```diff
@@ -187,8 +187,10 @@ private CommandSpec CreateCommandSpecUsingMuxerIfPortable(
         var depsFileArguments = GetDepsFileArguments(depsJsonFile);
         var additionalProbingPathArguments = GetAdditionalProbingPathArguments();
 
-        var muxerArgs = new List<string>();
-        muxerArgs.Add("exec");
+        var muxerArgs = new List<string>
```

Comment:
This is the one case in which I don't really like the "simpler" version, but I don't care that much

Reply:
Yep, exactly. To me, this looks worse than the original. Fewer total characters, but it takes up extra lines and means I need to consider a slightly less common code pattern. ["exec"] is concise and clear: a definite improvement.

Reply:
Fixed.